app/controllers/crime_applications_controller.rb

```diff
@@ -29,4 +29,24 @@ def destroy
   end
 
   def confirm_destroy; end
+
+  private
+
+  def order_param
+    return 'created_at' unless ordering_param_allowed?
+
+    params[:order]
+  end
+
+  def ordering_param_allowed?
+    %w[created_at].include? params[:order]
+  end
+
+  def sort_param
+    params[:sort]&.to_sym || :desc
```

Comment:
I think we will need to whitelist this param as well, because otherwise you can pass whatever you want in it and will try to trigger the sorting, which may pose a risk.